junit5/core/src/main/java/org/jboss/arquillian/junit5/ArquillianExtension.java

```diff
@@ -190,4 +201,36 @@ private boolean isRunAsClient(ExtensionContext extensionContext) throws Exceptio
         manager.getAdaptor().fireCustomLifecycle(runModeEvent);
         return runModeEvent.isRunAsClient();
     }
+
+    @Override
+    public boolean supportsParameter(final ParameterContext parameterContext, final ExtensionContext extensionContext) throws ParameterResolutionException {
+        try {
+            final TestRunnerAdaptor adapter = getManager(extensionContext)
+                .getAdaptor();
+            final Object instance = extensionContext.getRequiredTestInstance();
+            final Method method = extensionContext.getRequiredTestMethod();
+            final AtomicBoolean supported = new AtomicBoolean();
+            // adapter.before(instance, method, () -> supported.set(arquillianParameterResolver.supportsParameter(parameterContext, extensionContext)));
+            // return supported.get();
+            return arquillianParameterResolver.supportsParameter(parameterContext, extensionContext);
```

Comment:
When running the in-container tests, nothing seems to be observing the `Before` event so the supported boolean is never set. However, we do need to invoke the `arquillianParameterResolver` for clients within the context.